README.md

```diff
@@ -6,20 +6,33 @@
 Hellbender
 ================
 
-The public parts of the next generation of GATK/Picard/Prometheus methods engine and tools.
+The public parts of the next generation of GATK/Picard methods engine and tools.
 
 
-To build, run `gradle install` (gradle 2.2.1 recommended).
+Requirements
+------------
+* Java 8
 
-To run the main program, run `build/install/hellbender/bin/hellbender`.
+* Gradle 2.2.1
+
+* R 3.1.3 
+
+
+Installation
+------------
+To build and run all tests, run `gradle check`. Test report is in `build/reports/tests/index.html`.
+
+To only build, run `gradle install`.
```

Comment:
It 2.3.9 works with the one exception that it now wants `gradle installDist` instead of `gradle install` or `gradle installApp`